### DISCUSSION TITLE
How can I enable tag symbols? - Conventions

### DISCUSSION THREAD

**Discussion by jbadwaik** [squidfunk/mkdocs-material#6822] [Q&A]
I want to enable tag symbols mentioned [here](https://squidfunk.github.io/mkdocs-material/conventions/#version) for my own mkdocs site. How do I do that?

**Comment by kamilkrzyskow** [comment]
Hello @jbadwaik,
please use the search feature, searching for `conventions`: https://github.com/squidfunk/mkdocs-material/discussions?discussions_q=conventions
Leads to the following discussion:
- https://github.com/squidfunk/mkdocs-material/discussions/6544

I will lock this discussion, so you can follow-up in the previous discussion, if you have more questions.
